src/snovault/resource_views.py

```diff
@@ -364,6 +366,27 @@ def item_view_raw(context, request):
     return context.properties
 
 
+@view_config(context=Item, permission='view_raw', request_method='GET', name='history')
+def item_view_history(context, request):
+    db = request.registry[DBSESSION]
+    props = db.query(storage.PropertySheet).filter(storage.PropertySheet.rid == context.uuid).all()
```

Comment:
This might be fine but I wonder if you can access this through the context (Resource) itself? It looks like Resource has `data` (https://github.com/IGVF-DACC/snovault/blob/326909ee204d58c1bd1ba5197c03c830ca81c413/src/snovault/storage.py#L494) which points to CurrentPropertySheet, which has `history` (https://github.com/IGVF-DACC/snovault/blob/326909ee204d58c1bd1ba5197c03c830ca81c413/src/snovault/storage.py#L475) which points to all the PropertySheets, with the `transaction` relationship (https://github.com/IGVF-DACC/snovault/blob/326909ee204d58c1bd1ba5197c03c830ca81c413/src/snovault/storage.py#L457) you use in end?

So something like `property_sheets = context.data.history`? 

Might not be worth monkeying with if this already works.